go/vt/wrangler/keyspace.go

```diff
@@ -388,12 +410,18 @@ func (wr *Wrangler) masterMigrateServedType(ctx context.Context, keyspace string
 	}
 
 	// We've reached the point of no return. Freeze the tablet control records in the source masters.
-	if err := wr.freezeMasterMigrateServedType(ctx, sourceShards); err != nil {
+	if err := wr.updateFrozenFlag(ctx, sourceShards, true); err != nil {
 		wr.cancelMasterMigrateServedTypes(ctx, sourceShards)
 		return err
 	}
 
 	// Phase 2
+	// Always setup reverse replication. We'll start it later if reverseReplication was specified.
+	// This will allow someone to reverse the replication later if they change their mind.
```

Comment:
Ah so even if you don't pass the `reverse_replication` flag you can still start vreplication and revert the migration?

Reply:
Yes :)

Reply:
In fact, you should't start vreplication yourself. You should just reissue `MigrateServedTypes` with `-reverse_replication`.